### ISSUE TITLE
Poor README.md

### ISSUE THREAD

**Issue by arocki7** [Semantic-Org/Semantic-UI-Docs#380]
I don't understand why it's really very hard for me to understand the process.

It could be simple and neat to understand.

**Comment by naveennamani** [comment]
yeah, I don't understand either, what is the thing with *ui* folder and *css* folder.

It would be so great if build html doc folder can be shared as a release, this would have saved my 4 hours.
